testing.rst

```diff
@@ -619,15 +619,58 @@ This allows you to create all types of requests you can think of:
     :ref:`framework.test <reference-framework-test>` option is enabled).
     This means you can override the service entirely if you need to.
 
-.. caution::
+Multiple requests in one test
+.............................
+
+After you send one request, subsequent ones will make the client reboot
+the kernel, recreating the container from scratch.
+This ensures that requests are "isolated" using "new" service objects,
+but can cause some unexpected behaviors. For example, the security token will
+be cleared, Doctrine entities will be "detached"…
+
+Calling the client's
+:method:`Symfony\\Bundle\\FrameworkBundle\\KernelBrowser::disableReboot`
+method is the first step to work around this, as this will reset the kernel
+instead of rebooting it. Now, resetting the kernel will call the reset method
+of every ``kernel.reset`` tagged service, which will **also** clear the
+security token, detach entities and so on.
+
+As such, the next step is to create a
```

Comment:
```suggestion
As such, a necessary step is to create a
```

Using "next step" feels weird to me as there is no other step in this section

Reply:
The previous step is the previous paragraph (and is mentioned as such); is that not clear enough? :thinking:

Reply:
Ah yes that's right, my bad 👍